e2e/Microsoft.Azure.Devices.E2ETests/TwinE2ETests.cs

```diff
@@ -661,5 +728,84 @@ static public void ClassCleanup()
             TestContext.WriteLine("verified " + serviceTwin.Properties.Reported[propName].ToString() + "=" + propValue);
             await TestUtil.RemoveDeviceAsync(deviceInfo.Item1, registryManager);
         }
+
+        private async Task _Twin_ServiceDoesNotCreateNullPropertyInCollection(Client.TransportType transport)
+        {
+            var propName1 = Guid.NewGuid().ToString();
+            var propName2 = Guid.NewGuid().ToString();
+            var propEmptyValue = "{}";
+
+            Tuple<string, string> deviceInfo = TestUtil.CreateDevice(DevicePrefix, hostName, registryManager);
+            var deviceClient = DeviceClient.CreateFromConnectionString(deviceInfo.Item2, transport);
```

Comment:
`DeviceClient` is disposable and so it should be wrapped within a `using` statement:

```C# 
using (var deviceClient = DeviceClient.CreateFromConnectionString(deviceInfo.Item2, transport))
{
     // <test code>
}
```

Reply:
Added